File: Java/03. Code Related to Arrays/63. Find the longest Palindrome in an array.java
import java.util.*;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter the number of elements of the Array: ");
        int size = userInput.nextInt();
        int[] array = new int[size];

        for (int i = 0; i < size; i++) {
            System.out.print("Element " + (i + 1) + ": ");
            array[i] = userInput.nextInt();
        }

        int length = 0;

        for (int i : array) {
            String s = String.valueOf(i);
            String reverse = new StringBuilder(s).reverse().toString();
            if (s.equals(reverse)) {
                if (s.length() > String.valueOf(length).length()) {
                    length = i;
                }
            }
        }
        System.out.println(length);
        userInput.close();
    }
}

File: Python/03. Code Related to Arrays/63. Find the longest Palindrome in an array.py
lengthOfArray = int(input("Enter the length of an Array:"))
array = []

for i in range(lengthOfArray):
    element = input(f"Enter {i+1} element :")
    array.append(element)
l = 0
for i in array:
    s=str(i)
    if s==s[::-1]:
        if len(s)>len(str(l)):
            l=i
print(l)